package.json

```diff
@@ -67,6 +67,7 @@
 		"php:compatibility": "composer phpcs:compatibility",
 		"php:lint": "composer phpcs:lint",
 		"reformat-files": "yarn prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,json}\"",
+		"test-cli": "mocha tools/cli/tests/**/*.test.js --require esm",
```

Comment:
Moving the convo started earlier: https://github.com/Automattic/jetpack/pull/18195#issuecomment-766997465

this command won't work as-is. As you mentioned, cli was developed as a separate package. So I guess it would make sense to keep this command only in cli-specific `package.json`. Since this command would be useful for package developer, not a monorepo consumer.

Reply:
> Do y'all think it would be a better approach to not try to dev this as a separate project? 

Personally, I like the idea of a separate project. Fits well into a monorepo concept. 

TBH, I'm fine running it as `yarn jetpack`, and we also can provide a way to install/link the cli globally

Reply:
When I first pushed the branch, it was before the repo reorg, so mocha was still here because of Jetpack. Removing this line in d4e13dd05